packages/e2e-tests/src/specs/editor/prepublishChecklist.js

```diff
@@ -17,17 +17,24 @@
 /**
  * External dependencies
  */
-import { createNewStory, publishStory } from '@web-stories-wp/e2e-test-utils';
+import {
+  createNewStory,
+  publishStory,
+  triggerHighPriorityChecklistSection,
+  withExperimentalFeatures,
+} from '@web-stories-wp/e2e-test-utils';
 import percySnapshot from '@percy/puppeteer';
 
 describe('Pre-Publish Checklist', () => {
+  withExperimentalFeatures(['enableChecklistCompanion']);
```

Comment:
Not technically needed since the experiment is now enabled by default.

Reply:
Awesome, I'll remove it!